firstvoices/backend/views/sites_views.py

```diff
@@ -32,7 +34,7 @@
         description="Basic information about a language site, for authorized users.",
         responses={
             200: inline_site_doc_detail_serializer,
-            403: OpenApiResponse(description=doc_strings.error_403),
+            401: OpenApiResponse(description=doc_strings.error_401),
```

Comment:
No need to add 401 to the response list here. We list 401 as a possibility for all APIs at the top of the docs, along with 500 and a few others.